app/helpers/application_helper.rb

```diff
@@ -74,6 +74,9 @@ def colored_span(color, content)
   end
 
   def easter_check
-    Holidays.on(Date.today, :nl)[0][:name] == "Pasen"
+    holiday = Holidays.on(Date.today, :nl)
+    if holiday.any?
+      holiday[0][:name] == "Pasen"
+    end
```

Comment:
This could be a oneliner
And this method name could be something like `is_easter?` At least with the question mark so you know it'll return a boolean